textproc/libxslt/files/dynamic_lookup-11.patch

```diff
@@ -11,14 +11,14 @@ https://debbugs.gnu.org/cgi/bugreport.cgi?bug=44605
 -      # to the OS version, if on x86, and 10.4, the deployment
 -      # target defaults to 10.4. Don't you love it?
 -      case ${MACOSX_DEPLOYMENT_TARGET-10.0},$host in
--	10.0,*86*-darwin8*|10.0,*-darwin[91]*)
+-	10.0,*86*-darwin8*|10.0,*-darwin[912]*)
 -	  _lt_dar_allow_undefined='$wl-undefined ${wl}dynamic_lookup' ;;
 -	10.[012][,.]*)
 +    darwin*)
 +      case ${MACOSX_DEPLOYMENT_TARGET},$host in
-+	10.[012],*|,*powerpc*)
```

Comment:
Most of the original configure is in spaces, this one part has tabs, but in an inconsistent mix from line to line and within the same line. I used spaces everywhere not realizing there were tabs, the shell is OK with it

Reply:
Right, whether you use tabs or spaces does not matter to a shell.

They probably used the convention that a tab is intended to expand visually to 8 spaces. I don't personally like this style because it forces you to configure your editor to display tabs as 8 spaces but some editors can save files with that style of whitespace automatically and some developers like those of autoconf apparently use that.